frontend/src/pages/instance/Editor/index.vue

```diff
@@ -80,6 +84,8 @@ import ResizeBar from './components/drawer/ResizeBar.vue'
 export default {
     name: 'InstanceEditor',
     components: {
+        DashboardLink,
+        FfTabs,
```

Comment:
No need to include `FfTabs` here as it's a global component (as are all `/ui-components`)